CHANGELOG.md

```diff
@@ -1,17 +1,22 @@
 # Change Log
 
-## 0.6 (unreleased)
+## v0.6
 
 ### Summary
 
-- [#174](https://github.com/awslabs/amazon-s3-find-and-forget/pull/174): Add
-  support for types nested in struct
 - [#173](https://github.com/awslabs/amazon-s3-find-and-forget/pull/173): Show
   column types and hierarchy in the front-end during Data Mapper creation
 - [#173](https://github.com/awslabs/amazon-s3-find-and-forget/pull/173): Add
   support for char, smallint, tinyint, double, float
+- [#174](https://github.com/awslabs/amazon-s3-find-and-forget/pull/174): Add
+  support for types nested in struct
 - [#177](https://github.com/awslabs/amazon-s3-find-and-forget/pull/177):
   Reformat of Python source code (non-functional change)
+- Dependency version updates for:
+  - [#178](https://github.com/awslabs/amazon-s3-find-and-forget/pull/178),
```

Comment:
```suggestion
  - [#178](https://github.com/awslabs/amazon-s3-find-and-forget/pull/178) lodash,
```

Reply:
This is intended to have "number, number library" since both PR references are for the same library